lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -827,6 +827,86 @@ def add_payment_method(parameters, credit_card_or_vault_id, options)
           end
         end
       end
+
+      def bank_account_errors(payment_method, options)
+        if payment_method.validate.present?
+          payment_method.validate
+        elsif options[:billing_address].blank?
+          'billing_address is required parameter to store and verify Bank accounts.'
+        elsif options[:ach_mandate].blank?
+          'ach_mandate is a required parameter to process bank acccount transactions see (https://developer.paypal.com/braintree/docs/guides/ach/client-side#show-required-authorization-language)'
+        end
+      end
+
+      def add_customer_with_payment_method(payment_method, options)
+        if payment_method.is_a? Check
+          create_customer_from_payment_method(payment_method, options)
+          add_bank_account_to_customer(payment_method, options)
```

Comment:
I can't quite tell, but it seems like this would create two `Response`s outside of the context of a `MultiResponse`. I think this _might_ mean that the first response wouldn't be captured in the final transcript? Which ideally would be there for debugging purposes.

Reply:
Already covered on the refactor 👍